# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../runit/build-svwait.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./package/commands.orig	2008-06-15 22:53:26.000000000 +0200
+++ ./package/commands	2008-08-08 08:28:20.000000000 +0200
@@ -6,4 +6,6 @@
 runsvdir
 sv
 svlogd
+svwaitdown
+svwaitup
 utmpset
--- ./src/Makefile.orig	2008-06-15 22:53:26.000000000 +0200
+++ ./src/Makefile	2008-08-08 08:33:11.000000000 +0200
@@ -1,4 +1,4 @@
-IT=chpst runit runit-init runsv runsvchdir runsvdir sv svlogd utmpset
+IT=chpst runit runit-init runsv runsvchdir runsvdir sv svlogd svwaitup svwaitdown utmpset
 
 default: sysdeps $(IT)
 
